packages/moonstone/MoonstoneDecorator/fontGenerator.js

```diff
@@ -30,11 +30,13 @@ const fontName = 'Moonstone';
 // };
 const fonts = {
 	'ja': {
-		regular: 'LG Smart UI JP',
-		bold: 'LG Display_JP_Bold'
+		regular: 'LG Smart UI JP'
```

Comment:
According to Japan office, bold JP font is made for subtitle and it is not recommended to use for UI.